gobblin-data-management/src/main/java/gobblin/data/management/retention/dataset/SimpleDataset.java

```diff
@@ -0,0 +1,55 @@
+package gobblin.data.management.retention.dataset;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import gobblin.data.management.retention.policy.RetentionPolicy;
+import gobblin.data.management.retention.policy.TimeBasedRetentionPolicy;
+import gobblin.data.management.retention.version.TimestampedDatasetVersion;
+import gobblin.data.management.retention.version.finder.ModDateTimeDatasetVersionFinder;
+import gobblin.data.management.retention.version.finder.VersionFinder;
+
+
+/**
+ * {@link DatasetBase} for a simple dataset.
+ *
+ * Uses a {@link ModDateTimeDatasetVersionFinder} and a {@link TimeBasedRetentionPolicy}.
+ */
+public class SimpleDataset extends DatasetBase<TimestampedDatasetVersion> {
```

Comment:
Can we call this something different from SimpleDataset? (not very informative). Maybe ModificationTimeDataset?